### PR TITLE
Add tag to Gerrit review (#172)

### PR DESCRIPTION
**Pull request by SpOOnman**
Closes #172

### CODE REVIEW

File: src/main/java/pl/touk/sputnik/connector/gerrit/ReviewInputBuilder.java [hunk 1]
```diff
@@ -14,13 +16,16 @@
...
         ReviewInput reviewInput = new ReviewInput();
         reviewInput.message = Joiner.on(". ").join(review.getMessages());
-        reviewInput.labels = new HashMap<String, Short>(review.getScores());
-        reviewInput.comments = new HashMap<String, List<ReviewInput.CommentInput>>();
+        reviewInput.labels = new HashMap<>(review.getScores());
+        if (StringUtils.isNotBlank(tag)) {
+            reviewInput.tag = tag;
+        }
+        reviewInput.comments = new HashMap<>();
         for (ReviewFile file : review.getFiles()) {
```
Comment (alien11689) [inline src/main/java/pl/touk/sputnik/connector/gerrit/ReviewInputBuilder.java]: convert to streams

File: src/main/java/pl/touk/sputnik/connector/gerrit/ReviewInputBuilder.java [hunk 2]
```diff
@@ -14,13 +16,16 @@
...
-            List<ReviewInput.CommentInput> comments = new ArrayList<ReviewInput.CommentInput>();
+            List<ReviewInput.CommentInput> comments = new ArrayList<>();
             for (Comment comment : file.getComments()) {
```
Comment (alien11689) [inline src/main/java/pl/touk/sputnik/connector/gerrit/ReviewInputBuilder.java]: convert it to streams

File: src/test/java/pl/touk/sputnik/connector/gerrit/ReviewInputBuilderTest.java [hunk 3]
```diff
@@ -11,18 +11,41 @@
...
         assertThat(reviewInput.message).isEqualTo("Total 8 violations found");
         assertThat(reviewInput.comments).hasSize(4);
+        assertThat(reviewInput.tag).isEqualTo(TAG);
         assertThat(reviewInput.comments.get("filename1")).hasSize(2);
         assertThat(reviewInput.comments.get("filename1").get(0).message).isEqualTo("test1");
         assertThat(reviewInput.labels.get("Code-Review")).isEqualTo((short) 1);
     }
 
+    @Test
+    void shouldNotSetEmptyTag() {
```
Comment (alien11689) [inline src/test/java/pl/touk/sputnik/connector/gerrit/ReviewInputBuilderTest.java]: this and next test should be one parameterized test

File: README.md [hunk 4]
```diff
@@ -22,6 +22,8 @@ connector.port=8080
 connector.username=sputnik
 connector.password=PassWd
 connector.useHttps=false
+connector.verifySsl=false
+connector.tag=sputnik
```
Comment (alien11689) [inline README.md]: Can you add some description for new parameter in README?

File: src/main/java/pl/touk/sputnik/connector/gerrit/ReviewInputBuilder.java (hunk 1 above)
Comment (SpOOnman) [reply src/main/java/pl/touk/sputnik/connector/gerrit/ReviewInputBuilder.java]: Done

File: src/main/java/pl/touk/sputnik/connector/gerrit/ReviewInputBuilder.java (hunk 2 above)
Comment (SpOOnman) [reply src/main/java/pl/touk/sputnik/connector/gerrit/ReviewInputBuilder.java]: Done

File: src/test/java/pl/touk/sputnik/connector/gerrit/ReviewInputBuilderTest.java (hunk 3 above)
Comment (SpOOnman) [reply src/test/java/pl/touk/sputnik/connector/gerrit/ReviewInputBuilderTest.java]: Done with @NullAndEmptySource

File: README.md (hunk 4 above)
Comment (SpOOnman) [reply README.md]: I need to work on README in next PR. There are lots of options not exposed nor described there.

**Pull request merged**